pkg/clients/dynatrace/settings_logmonitoring.go

```diff
@@ -0,0 +1,127 @@
+package dynatrace
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube/logmonitoring"
+	"github.com/Dynatrace/dynatrace-operator/pkg/clients/utils"
+)
+
+type IngestRuleMatchers struct {
+	Attribute string   `json:"attribute,omitempty"`
+	Operator  string   `json:"operator,omitempty"`
+	Values    []string `json:"values,omitempty"`
+}
+
+type Value struct {
+	Title    string               `json:"config-item-title,omitempty"`
+	Matchers []IngestRuleMatchers `json:"matchers,omitempty"`
+	Enabled  bool                 `json:"enabled,omitempty"`
+	Storage  bool                 `json:"send-to-storage,omitempty"`
+}
+
+type Item struct {
+	ObjectID string `json:"objectId,omitempty"`
+	Val      Value  `json:"value,omitempty"`
+}
```

Comment:
we are in the big dtclient package, so these names are a bit too general (also do they need to be public?)

you can solve this 2 ways:
1. Come up with more specific names (will also help with the naming of `Item.Val`)
2. I saw a pattern that you could use (we don't really do it)
   - You can define `structs` within a function, so it can be only used in it
   - This is useful when you only need the struct to create the body for a POST request
   - So you just use it in a function to create a `[]byte` using `json.Marshal`, and then just pass it around as a `[]byte`
      - This is nice to not pollute the package with throw-away types. 
      -  Be careful, as not all types may be that `throw-away`

Reply:
Either way is fine by me. Way 1. is simpler and faster, but 2. is probably the "proper way".

However, we might want to look into using the 2. way all around the dtclient. (or to just restructure it a bit, as it's ever growing)

Reply:
I would gladly try to do 2. But then again, wouldn't this be better a whole refactoring story, where we update the whole client package? (I am not sure how big of a change this would be)

Reply:
Just do option 1. for now
And create a follow up for possible improvements for option 2.

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/4053/commits/98b868536e9d5168d1d3db5548f1d2afa67eef45

Also: I needed to alter the maxDescLen for our manifests a little bit because it got too large. I do not know why I did not get this error before.